src/jobs/README.md

```diff
@@ -303,7 +303,7 @@ jobs:
       context: dev
       path: "myapp"
       isopod_config: "isopod.yml"
-      isopod_version: "0.29.6"
+      isopod_version: "0.30.0"
```

Comment:
Isopod version is repeated multiple times. Maybe you want to extract it in y[aml anchor](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/)

Reply:
this is the readme - does not really make sense